specification/security/resource-manager/Microsoft.Security/preview/2023-07-01-preview/examples/integrations/DeleteIntegration_example.json

```diff
@@ -0,0 +1,15 @@
+{
+  "parameters": {
+    "api-version": "2023-07-01-preview",
+    "scope": "subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+    "integrationId": "ad9a8e26-29d9-4829-bb30-e597a58cdbb8"
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "Location": "https://management.azure.com/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/integrationId/ad9a8e26-29d9-4829-bb30-e597a58cdbb8?api-version=2024-01-31"
```

Comment:
This example is incorrect:
Location should use the api-version 2023-07-01-preview and not 2024-01-31
The /providers/Microsoft.Security/integrations/{integrationId} is missing (path is incorrect) 
The response code in the example is 202, but the swagger spec for single GET is 200